roles/variables/vars/main.yml

```diff
@@ -1464,14 +1465,14 @@ ksql_properties:
       public.key.path: "{{rbac_enabled_public_pem_path}}"
       confluent.metadata.bootstrap.server.urls: "{{mds_bootstrap_server_urls}}"
   rbac_ldap:
-    enabled: "{{rbac_enabled and ('ldap' in auth_mode) and (not ksql_mds_cert_auth_only) }}"
+    enabled: "{{ rbac_enabled and auth_mode == 'ldap' and (not ksql_mds_cert_auth_only) }}"
```

Comment:
why did we change it from dual to ldap only? is it not required in ldap+oauth?

Reply:
these are ksql as mds client properties. 
for all other components whenever we have ldap_with_oauth on mds we use oauth for client-mds communication. In ksql we did it differently i dont recall why. So making it similar to other components should be fine imv

Reply:
Yes Me and Mansi had this conversation yesterday. We are aligned on this.

Reply:
One question here. In case someone upgrades from 7.7/7.8 to 7.9 and have auth_mode = ldap_with_oauth Then this would mean ksql to mds communication which was earlier happening over ldap will now happen on oauth. This is fine if their inventory file has oauth credentials for ksql. But if they dont have it then in such case their setup was working in 7.7/7.8 and after upgrading to 7.9 it might fail

Reply:
Yes good point. Will discuss this with Shreeda.

Reply:
Lets create a seperate ticket to address this issue with uprgade and merge the rest of code from this pr